### DISCUSSION TITLE
Backup, snapshot, and logging options

### DISCUSSION THREAD

**Discussion by inbytesdevteam** [supabase/supabase#488] [Questions] [answered]
What's the best way to backup your data in Supabase? 
Do you just save all your data locally at certain time? Or do you create a snapshot or something?

Any suggestions? Thanks :)

**Comment by inian** [comment, marked as answer]
Hi @inbytesdevteam, the database is backed up every day. You can find the backups and the time associated with the backups here - https://app.supabase.io/project/your-project-ref/database/backups

**Reply by inbytesdevteam** [reply]
Oh! I didn't know supabase has this feature! Thanks @inian. :)

**Reply by mstade** [reply]
It’s a great feature that’s saved us a couple of times already!

**Comment by thorwebdev** [comment]
On the pro tier you will get access to 7 days of backups (we take a snapshot every evening) - this is available on the Database > Backups page in the dashboard. It takes the form of a .sql file you can download and use to restore your entire schema + data.
​
We're currently working on a point in time backup option for user's on the usage based tier so you can restore from any moment within the last 30 days but this is still in development (see pricing here: https://supabase.io/pricing).

You are also free to setup your own custom backup mechanism if you require, since the db is just plain old postgres, pg_dump from the command line will work out of the box with the connection string provided in the dashboard.

For transaction logging, again it is just plain postgres, so pg_stat_statements, pgAudit, are all great solutions. We are working on offering this as an added service inside the dashboard itself so you can explore the API and DB logs directly, but it will take some weeks before this is launched.

We do currently surface Auth logs for audit purposes in the Auth > Settings page if this is of interest.
